presto-hive/src/main/java/com/facebook/presto/hive/security/SystemTableAwareAccessControl.java

```diff
@@ -26,17 +26,16 @@
 import java.util.Optional;
 import java.util.Set;
 
-import static com.facebook.presto.hive.HiveMetadata.getSourceTableNameForPartitionsTable;
-import static com.facebook.presto.hive.HiveMetadata.isPartitionsSystemTable;
+import static com.facebook.presto.hive.HiveMetadata.getSourceTableNameFromSystemTable;
 import static com.facebook.presto.spi.security.AccessDeniedException.denySelectTable;
 import static java.util.Objects.requireNonNull;
 
-public class PartitionsAwareAccessControl
+public class SystemTableAwareAccessControl
         extends ForwardingConnectorAccessControl
 {
     private final ConnectorAccessControl delegate;
 
-    public PartitionsAwareAccessControl(ConnectorAccessControl delegate)
+    public SystemTableAwareAccessControl(ConnectorAccessControl delegate)
```

Comment:
Curious why this part of change is missing in this file: https://github.com/prestosql/presto/commit/e7478a3200c26f9ca312f9b29c752de4e4843b7e#diff-a8c5fae2119a67c33cf42d8306cd2f16R70-R72 -- just want to double check if it's not accidentally missing :)

Reply:
not missing. Some difference between the two files